task02_iryna/src/dao/Paragraph.java

```diff
@@ -0,0 +1,36 @@
+package dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Paragraph implements CompositeWord{
+
+    public static int sentensesCounter;
```

Comment:
Class variable fields should not have public accessibility. By using private attributes and accessor methods (set and get), unauthorized modifications are prevented.